src/pages/LogIn/LogInForm/container.tsx

```diff
@@ -37,12 +38,20 @@ const LogInFormContainer: FunctionComponent<ILogInFormContainerProps> = ({ class
 
   const onSubmit = async (
     values: ILogInFormValues,
-    { setSubmitting }: FormikHelpers<ILogInFormValues>
+    { setSubmitting, setErrors }: FormikHelpers<ILogInFormValues>
   ) => {
-    const { data } = await login({ variables: values });
-    Session.login(data.login);
-    setSubmitting(false);
-    history.push(RoutesBuilder.applicant.home());
+    try {
+      const { data } = await login({ variables: values });
+      Session.login(data.login);
+      setSubmitting(false);
+      history.push(RoutesBuilder.applicant.home());
+    } catch (e) {
```

Comment:
acotaría el cuerpo del try a la linea que puede pinchar

Reply:
mmm esto depende del issue de manejo de errores, pero aca un error por ejemplo de could not connect to server va a mostrar un mensaje en los fields

tambien hay que ver qué pasa en prod, donde probablemente las excepciones del back con su backtrace y message no pasan al front 🤔

Reply:
Pero no quiero ejecutrar las tres lineas debajo de la mutation si falla

Reply:
pondria un return en el catch, los try tienen que ser lo mas chicos posibles, sobre todo cuando catchean todo tipo de error